backend/packages/Upgrade/src/api/services/FeatureFlagService.ts

```diff
@@ -204,6 +206,23 @@ export class FeatureFlagService {
     return this.segmentService.deleteSegment(segmentId, logger);
   }
 
+  public async validate(flagDTO: FeatureFlagValidation, logger: UpgradeLogger) {
```

Comment:
`validate` is vague, we are specifically validating key + context uniqueness.

Reply:
can this be changed to `validateUniqueAppContextKey` or some such thing?

Reply:
name refractored